CooperationHullMainSite/Controllers/HomeController.cs

```diff
@@ -138,8 +138,8 @@ public IActionResult TheBigIdea()
 
 
         [HttpGet]
-        [Route("ToolsOfTheTrade")]
-        public IActionResult ToolsOfTheTrade()
+        [Route("HowWeWork")]
```

Comment:
Is there any way we could sluggify these pages? so it would be `/how-we-work` instead of `/HowWeWork`?

Reply:
Should be a simple change, I used our work naming convention by default with really thinking about it.
If you add a task for it I'll pick it up at some point.

Reply:
added